tsp-output/@azure-tools/typespec-csharp/src/Generated/Models/CreateChatCompletionRequest.Serialization.cs

```diff
@@ -258,16 +258,34 @@ void IJsonModel<CreateChatCompletionRequest>.Write(Utf8JsonWriter writer, ModelR
                 foreach (var item in _serializedAdditionalRawData)
                 {
                     writer.WritePropertyName(item.Key);
+
+                    switch (item.Value)
```

Comment:
Changes to how we write _serializedAdditionalRawData, to support extension properties (for input models).